doc/release/0.18.0-notes.rst

```diff
@@ -65,18 +70,124 @@ zero-phase low-pass FIR filter, and downsamples using `scipy.signal.upfirdn`.
+expect that this change will improve numeric stability of optimization in the
+vast majortiy of use cases. See pull request #5536 for details.
+
+Handling of infinite bounds in SLSQP optimization has been improved. We expect
+that this change will improve numeric stability of optimization in the vast
+majortiy of use cases. See pull request #6024 for details.
+
+A large suite of global optimization benchmarks has been added to 
+``scipy/benchmarks/go_benchmark_functions``. See pull request #4191 for details.
+
+
+`scipy.stats` improvements
+--------------------------
+
+Trapezoidal distribution has been implemented as `scipy.stats.trapz`.
+Skew normal distribution has been implemented as `scipy.stats.skewnorm`.
+Burr type XII distribution has been implemented as `scipy.stats.burr12`. 
+
+Random matrices
+~~~~~~~~~~~~~~~
+
+`scipy.stats.special_ortho_group` and `scipy.stats.ortho_group` provide
+generators of random matrices in the SO(N) and O(N) groups, respectively. They
+generate matrices in the Haar distribution, the only uniform distribution on
+these group manifolds.
+
+`scipy.stats.random_correlation` provides a generator for random
+correlation matrices, given specified eigenvalues.
+
+
+`scipy.linalg` improvements
+---------------------------
+
+`scipy.linalg.svd` gained a new keyword argument, ``lapack_driver``. Available
+drivers are ``gesdd`` (default) and ``gesvd``.
+
+`scipy.linalg.lapack.ilaver` returns the version of the LAPACK library SciPy
+links to.
+
+
+`scipy.spatial` improvements
+----------------------------
+
+Boolean distances, `pdist`, has been sped up. Improvements vary by the function
+and the input size. In many cases, one can expect a speed-up of x2--x10. 
+
+New class `scipy.spatial.SphericalVoronoi` constructs Voronoi diagrams on the
+surface of a sphere. See pull request #5232 for details.
+
+`scipy.cluster` improvements
+----------------------------
+
+New clustering algorithm, the nearest neighbor chain algorithm, has been
+implemented for `scipy.cluster.hierarchy.linkage`. As a result, one can expect
+a significant algorithmic improvement for several linkage methods. 
+
+
+`scipy.special` improvements
+----------------------------
+
+`scipy.special.loggamma` computes the principal branch of the logarithm of the
+Gamma function. For real input, ``loggamma`` is compatible with
+`scipy.special.gammaln`. For complex input, it has more consistent behavior in
+the complex plane and should be preferred over ``gammaln``.
+
+Vectorized forms of spherical Bessel functions have been implemented as 
+`scipy.special.spherical_jn`, `scipy.special.spherical_kn`,
+`scipy.special.spherical_in` and `scipy.special.spherical_yn`.
+They are recommended for use over ``sph_*`` functions, which are now deprecated.
+
+Several special functions have been extended to the complex domain and/or
+has seen domain/stability improvements. This includes `spense`, `digamma`,
```

Comment:
spense -> spence and has seen -> have seen; other than that (FWIW) special looks good to me.